sway-fmt/src/fmt.rs

```diff
@@ -641,4 +641,21 @@ fn one_liner() -> bool {
         let (_, formatted_code) = result.unwrap();
         assert_eq!(correct_sway_code, formatted_code);
     }
+
+    #[test]
+    // Test that the use statements with multiple imports are properly formatted
+    fn test_use_statement() {
+        let expected_sway = r#"script;
+use std::chain::{panic,log_u8};
```

Comment:
Although unrelated to this fix, a new issue could be created to actually add space between two imported values.

Current
```
use std::chain::{panic,log_u8};
```
Expected
```
use std::chain::{panic, log_u8};
```

Reply:
Good call. New issue pls (and probably a `good first issue`!).

Reply:
New issue created!

Reply:
Ref: #777